src/app/assets/languages/fr.json

```diff
@@ -0,0 +1,1011 @@
+      "no-permissions-found": "Aucune autorisation trouv\u00e9e",
+      "no-filtered-permissions-body": "Aucune autorisation ne correspond \u00e0 vos crit\u00e8res de recherche.",
+      "no-caches-found": "Aucune cache trouv\u00e9e",
+      "no-filtered-caches-body": "Aucun cache ne correspond \u00e0 vos crit\u00e8res de recherche.",
+      "permissions-implicit-warning": "Les permissions pour les r\u00f4les pr\u00e9d\u00e9finis ne sont pas modifiables.",
+      "update-success": "Le r\u00f4le {{name}} a \u00e9t\u00e9 mis \u00e0 jour",
+      "update-error": "Erreur inattendue mettant \u00e0 jour le r\u00f4le {{name}}",
+      "modal-add-permission-title": "Ajouter des permissions",
+      "modal-remove-permission-title": "Supprimer l'autorisation {{name}}",
+      "modal-remove-permission-description-1": "Voulez-vous vraiment supprimer cette autorisation du r\u00f4le {{roleName}}? Apr\u00e8s la suppression, l'acc\u00e8s au r\u00f4le sera modifi\u00e9.",
+      "modal-remove-permission-description-2": "Vous pouvez toujours ajouter la permission de r\u00f4le \u00e0 nouveau."
+    },
+    "principals": {
+      "principal-name": "Nom principal du cartographe",
+      "principal-name-tooltip": "Pour permettre l'autorisation, les principaux doivent \u00eatre cartographi\u00e9s en noms de r\u00f4les, qui sont ensuite \u00e9tendus \u00e0 un ensemble de permissions.",
+      "roles": "R\u00f4les",
+      "search-placeholder": "Rechercher par nom",
+      "no-principals-found": "Aucun directeur n'a \u00e9t\u00e9 trouv\u00e9",
+      "no-principals-body": "Vous pouvez accorder l'acc\u00e8s \u00e0 un mapper principal.",
+      "no-filtered-principals-body": "Aucun principe ne correspond \u00e0 vos crit\u00e8res de recherche.",
+      "principals-hint-link": "En savoir plus dans la documentation {{brandname}}",
+      "loading-principals": "Chargement des principaux",
+      "loading-principals-error": "Erreur impr\u00e9vue lors du chargement des principaux",
+      "grant-access": "Accorder un acc\u00e8s",
+      "no-principals-status": "Pas de cartographic principal",
+      "modal-grant-title": "Accorder un nouvel acc\u00e8s",
+      "modal-grant-description": "Accorder l'acc\u00e8s \u00e0 {{brandname}} \u00e0 un mandant en accordant des r\u00f4les.",
+      "modal-principal-name": "Directeur",
+      "modal-principal-name-is-required": "Le nom du mappeur de r\u00f4le principal est requis",
+      "modal-roles-is-required": "Choisir au moins un r\u00f4le",
+      "modal-principal-exists": "Le r\u00f4le principal de la carte {{name}} existe d\u00e9j\u00e0",
+      "modal-user-exists": "Un utilisateur nomm\u00e9 {{name}} existe. Vous n'avez pas besoin de cr\u00e9er un mapper principal.",
+      "modal-roles": "R\u00f4les",
+      "modal-roles-list-placeholder": "S\u00e9lectionner les r\u00f4les",
+      "grant-success": "Acc\u00e8s accord\u00e9 \u00e0 {{112,114,105,110,99,105,112,97,108,78,97,109,101}.",
```

Comment:
The CSV of numbers inbetween `{{ }}` is a bug in my script. I'll fix this if you think the previous translations are reasonable, otherwise I'll just close the PR.